module/documents/active-effect.mjs

```diff
@@ -511,8 +511,23 @@ export default class ActiveEffect5e extends ActiveEffect {
     const actor = canvas.hud.token.object?.actor;
     if ( !actor ) return;
 
-    if ( target.dataset?.statusId === "exhaustion" ) ActiveEffect5e._manageExhaustion(event, actor);
-    else if ( target.dataset?.statusId === "concentrating" ) ActiveEffect5e._manageConcentration(event, actor);
+    const id = target.dataset?.statusId;
+    if ( id === "exhaustion" ) ActiveEffect5e._manageExhaustion(event, actor);
+    else if ( id === "concentrating" ) ActiveEffect5e._manageConcentration(event, actor);
+
+    if ( target.classList.contains("active") ) return;
+
+    const riders = CONFIG.statusEffects.find(e => e.id === id)?.riders;
+    if ( !riders ) return;
+
+    const createRider = async id => {
+      const existing = actor.effects.get(staticID(`dnd5e${id}`));
+      if ( existing ) return;
+      const effect = await ActiveEffect.implementation.fromStatusEffect(id);
+      return ActiveEffect.implementation.create(effect, { parent: actor, keepId: true });
+    };
+
+    Promise.all(Array.from(riders).map(createRider));
```

Comment:
This doesn't handle other avenues of condition application, such as via the character sheet or enrichers. It can probably be moved to `ActiveEffect5e#_onCreate` though, I think that should cover all cases.

Reply:
Shows how often I look at the Effects tab.